src/app/legacy/psammead-story-promo/src/index.jsx

```diff
@@ -32,23 +38,56 @@ import { grid } from '#legacy/psammead-styles/src/detection';
 import ImageGridItem from './ImageStyles';
 import TextGridItem from './TextStyles';
 
-const PROMO_TYPES = oneOf(['top', 'regular', 'leading']);
+const PROMO_TYPES = oneOf(['top', 'regular', 'leading', 'onwardJourneys']);
 
-const wrapperTopStoryStyles = `
+const wrapperTopStyles = `
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     grid-template-columns: repeat(12, 1fr);
   }
 `;
 
+const wrapperOnwardJourneysStyle = ({ displayImage, displayTimestamp }) => `
+  background-color: ${C_WHITE};
```

Comment:
I think we'll need to determine this colour based on if its an Article or CPS page. Article pages will have a white background for the promo, whereas CPS will have a grey one I believe.

Reply:
Ah cool! Didn't notice that 😅  Do you think WSTEAM1-98 should be done first? Just thinking that the changes here look good, but on CPS pages the padding looks a little off on the OJ promos since the background colour of them is the same as the background colour of the page.

Reply:
Could do; but since it's still in shaping I am not sure how long we want to delay the ship of these styles. Let me ask Jon about it.

Reply:
Yea may not be a big issue if the next step is getting the main background colour change. I didn't realise that was happening so soon, so it'll be great to not have to check for page types or pass colours around!